src/containers/Books/Books.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import { useHistory } from "react-router-dom";
+import BookCard from '../../components/BookCard/BookCard';
+import PaginationButton from '../../components/PaginationButton/PaginationButton';
+import SearchPanel from '../../components/SearchPanel/SearchPanel';
+import Loader from '../../components/Loader/Loader';
+import { getBooks } from '../../API/fetch';
+
+function Books() {
+  const [data, setData] = useState([]);
+  const [resultsValue, setResultsValue] = useState(0);
+  const [bookTitle, setBookTitle] = useState('');
+  const [sort, setSort] = useState('');
+  const [category, setCategory] = useState('');
```

Comment:
so many states with one meaning, it will be cause of extra rendering

Reply:
I will think what can i do with it